File: CreatingaSplashScreenMediaPlayer/app/src/main/java/com/example/creatingasplashscreenmediaplayer/MainActivity.java
package com.example.creatingasplashscreenmediaplayer;

import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;
import android.os.Looper;

import androidx.appcompat.app.AppCompatActivity;

// Amber Lawson - GP Splash Screen & MediaPlayer - 2026-06-04
public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        // create a handler to pause before launching the main activity
        new Handler(Looper.getMainLooper()).postDelayed(() -> {
            Intent intent = new Intent(MainActivity.this, MainActivity2.class);
            startActivity(intent);
            finish();
        }, 10000);
    }
}

File: ContactsMobileApp/app/src/main/java/com/example/contactsmobileapp/MainActivity.kt
package com.example.contactsmobileapp

import android.content.Intent
import android.os.Bundle
import android.widget.Button
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity
import androidx.core.view.ViewCompat
import androidx.core.view.WindowInsetsCompat

// Amber Lawson
// PA - Contacts Mobile App
// 05/14/2026
class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        enableEdgeToEdge()
        setContentView(R.layout.activity_main)
        ViewCompat.setOnApplyWindowInsetsListener(findViewById(R.id.main)) { v, insets ->
            val systemBars = insets.getInsets(WindowInsetsCompat.Type.systemBars())
            v.setPadding(systemBars.left, systemBars.top, systemBars.right, systemBars.bottom)
            insets
        }

        findViewById<Button>(R.id.btnContact1).setOnClickListener {
            startActivity(Intent(this, ContactDetailActivity::class.java).apply {
                putExtra("name", getString(R.string.contact1_name))
                putExtra("phone", getString(R.string.contact1_phone))
                putExtra("email", getString(R.string.contact1_email))
                putExtra("address", getString(R.string.contact1_address))
            })
        }

        findViewById<Button>(R.id.btnContact2).setOnClickListener {
            startActivity(Intent(this, ContactDetailActivity::class.java).apply {
                putExtra("name", getString(R.string.contact2_name))
                putExtra("phone", getString(R.string.contact2_phone))
                putExtra("email", getString(R.string.contact2_email))
                putExtra("address", getString(R.string.contact2_address))
            })
        }
    }
}
